mod_modular_vanilla/hooks/entity/world/player_party.nut

```diff
@@ -19,14 +19,15 @@
 			this.m.Strength += this.MV_getEmptyBroStrength() * (::World.Assets.getBrothersScaleMin() - roster.len());
 		}
 
-		// sort descending
-		strengths.sort(@(_a, _b) -a <=> -_b);
-
-		for (local i = 0; i < strengths.len() && i < ::World.Assets.getBrothersScaleMax(); i++)
+		if (strengths.len() > ::World.Assets.getBrothersScaleMax())
```

Comment:
can be `else if` to improve performance slightly

Reply:
I thought about it but vanilla seems to check those two conditions without an `else` so I wanted to preserve that behavior. Even though logically it makes no sense that `getBrothersScaleMax()` would be smaller than `getBrothersScaleMin()` but vanilla, at least in this function, wrote the code in a way that will not break with that situation.

What do you think considering that?

Reply:
NO problem then. Wasnt a big issue thing anyways